cdap-watchdog/src/main/java/co/cask/cdap/logging/LoggingConfiguration.java

```diff
@@ -59,7 +60,7 @@
 
   public static final long DEFAULT_LOG_SAVER_EVENT_BUCKET_INTERVAL_MS = 1 * 1000;
   public static final long DEFAULT_LOG_SAVER_MAXIMUM_INMEMORY_EVENT_BUCKETS = 4;
-  public static final long DEFAULT_LOG_SAVER_INACTIVE_FILE_INTERVAL_MS = 60 * 60 * 1000;
+  public static final long DEFAULT_LOG_SAVER_MAX_FILE_LIFETIME_MS = TimeUnit.HOURS.toMillis(1);
```

Comment:
Should we add a comment somewhere saying how it needs to be related to the delegation token expiry time?